packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -797,7 +807,23 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                     this.pan(PAN_RIGHT);
                     break;
                 }
+                case 'Shift': {
+                    if (!this.isRightClick) {
+                        this.setState({ cursor: 'crosshair', shiftKey: true });
+                    }
+                    break;
+                }
+            }
+        }
+    }
+
+    private onKeyUp(key: React.KeyboardEvent) {
+        if (key.key === 'Shift') {
```

Comment:
Same question again, here for this key string value. Not sure if such duplication is done elsewhere or treated as normal (or expected).

Reply:
I would rather leave as is, as explained above.